mobile/src/Student/screens/StudentGameIntro/IntroInfo/index.js

```diff
@@ -1,18 +1,18 @@
 import React from 'react'
 import { StyleSheet, Text, View, Image, Dimensions } from 'react-native'
 import { scale, verticalScale } from "react-native-size-matters"
-import { fontFamilies, fonts, colors } from '../../../../utils/theme'
+import { fontFamilies, fonts, colors, fontWeights } from '../../../../utils/theme'
 const win = Dimensions.get('window');
 
-const IntroInfo = ({ screen, Monsters, phone, sticker, text }) => {
+const IntroInfo = ({ monsterStyles, Monsters, phone, sticker, text, showSticker }) => {
     return (
         <View style={styles.container}>
-            <Image source={Monsters[0]} style={screen === 1 ? styles.monster1 : styles.monster3} />
+            <Image source={Monsters[0]} style={monsterStyles[0]} />
```

Comment:
fyi: There's this concept called defensive coding which means you try to guard your code always against any failures and crashes, sometimes even going stream and guarding it against something that in theory should never happens.

This applies here to `Monsters` variable. Several notes on it:

- `Monsters` variable can be similar to others and follows camel-based notation, `monsters`
- Check `Monsters` length to make sure there are at least two items in it before accessing them. This can be done as part of adding assertions before `return (`

Reply:
Thanks, will definitely keep this in mind! Would it be worthwhile to add a try-catch statement, change to camel-case and repush?

Reply:
I'm ok with the current state, your choice if you'd like to do the changes and push or go with the existing one